### DISCUSSION TITLE
Cant use importer

### DISCUSSION THREAD

**Discussion by SaveliiMoshkota** [firefly-iii/firefly-iii#8166] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I am experiencing an issue with the importer sub-app in Firefly III while using Docker. After entering my client ID and pressing submit, I encounter a blank page at the following URL `https://app:8080/oauth/authorize?client_id=4&redirect_uri=http%3A%2F%2Flocalhost%3A81%2Fcallback&response_type=code&scope=&state=...&code_challenge=....&code_challenge_method=S256`. 

I suspected it might be related to the URL, so I attempted using localhost:81 as well.

### Debug information

my client id:
![image](https://github.com/firefly-iii/firefly-iii/assets/148792439/6548fff0-6e1e-4564-ab7a-bcd5e3075777)
screen-shots:
![image](https://github.com/firefly-iii/firefly-iii/assets/148792439/6f7bf3c3-85af-47e7-a97a-cc41dd611d9e)

### Expected behaviour

I expect to be redirected to the appropriate page after submitting the client ID.

### Steps to reproduce

Follow the instructions 
![image](https://github.com/firefly-iii/firefly-iii/assets/148792439/12d3e297-121a-443f-856d-71a1ff4a757d)

**Comment by JC5** [comment]
You need to set the vanity url.

https://docs.firefly-iii.org/data-importer/installation/docker/

**Reply by SaveliiMoshkota** [reply]
ow, thanks
Looks like I missed it
